pocld/common.hh

```diff
@@ -402,10 +402,11 @@ struct EventPair {
 std::string hexdigits(std::string, uint8_t);
 
 // Type for the buffer allocation identifier, a running number starting from
-// 0. Note: In some parts functions, the input can be declared as a buffer
-// id which might be treated as a raw 64b SVM pointer instead of a cl_mem
-// buffer identifier, in case referring to SVM buffers.
-typedef uint64_t BufferId_t;
+// 0 with 0 denoting a NULL/illegal buffer.
+// Note: Currently in some parts functions, the input can be declared as
+// a buffer id which might be treated as a raw 64b SVM pointer instead of
+// a cl_mem buffer identifier, in case referring to SVM buffers (WiP).
```

Comment:
Probably worth turning this into a doc comment (`/// ...` or `/** ... */`) while at it. Then it should get displayed in the LSP tooltip when hovering over the type name elsewhere in the code.